app/app/templates/data.html

```diff
@@ -13,6 +13,12 @@
 				<p>Data Analytics</p>
 			</a>
 		</li>
+		<li class="nav-item">
+			<a class="nav-link" href="{{ url_for('routes.export')}}">
+				<i class="material-icons">arrow_circle_down</i>
+				<p>Export Data</p>
+			</a>
+		</li>
```

Comment:
I think you explained the code repetition, we can refactor this, but of course not in this issue. Can you put a GitHub issue to defer the task?

Reply:
Issue is created